crates/context/src/lib.rs

```diff
@@ -323,22 +339,25 @@ impl ContextManager {
         }
 
         let handle = self.store.handle();
-
         let identity_key = ContextIdentityKey::new(context_id, invitee_id);
-
         if handle.has(&identity_key)? {
```

Comment:
these are complex pieces, let's not impair readability

Reply:
Reverted